README.md

```diff
@@ -11,7 +11,9 @@ HITOBITO_EMAIL=pfnörch@flamberg.ch
 HITOBITO_TOKEN=*******************
 HITOBITO_SERVER=https://db.scout.ch
 HITOBITO_LANG=de -- one of: de, fr, it
+HITOBITO_SENDER=1234
 ```
+`HITOBITO_SENDER` is the personal number of the person who will be the sender of the invoice, usually the treasurer. 
```

Comment:
we don't need this in the config (store per bulk by using the authenticated users id)

Reply:
Added clarification